Final Project Submission/MusicBox.cpp

```diff
@@ -0,0 +1,1289 @@
+// This program uses the Beep() and Sleep() functions to play simple songs that the user chooses from a list.
+// Beep() is used to produce sound, Sleep() is used for delays between notes in the case of rests.
+
+#include <iostream>
+#include <windows.h> // For Beep() and Sleep()
+#include <stdlib.h>
+
+/* Defining note frequencies in Hz. I decided to cover my bases and define frequencies chromatically,
+hence why I also include flats. For example, A4 = A, while Ab4 is 1 semitone below it, making Ab4 = A-flat/G-sharp.
+I decided to #define all their values rather than declare them as consts because, as I understand it, #define doesn't
+force the program to allocate memory for them. */
+#define C1 32
```

Comment:
#define is not desirable nor necessary. Allocating memory for a `const int` is not really a problem, and the compilers usually optimize them away regardless.